### PR TITLE
Replace six.iter* to python 3 equivalent

### PR DESCRIPTION
**Pull request by dorothykiz1**
fixes https://github.com/pandas-benchmarks-grant/board-issues/issues/2

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,5 +12,5 @@ jobs:
       - name: Install dependencies
         run: pip install flake8
       
-      - name: flake8
-        run: flake8 . 
\ No newline at end of file
+      - name: flake8 
+        run: flake8 .
```
Comment (datapythonista) [inline .github/workflows/ci.yml:16]: Can you have a look at this?

**Review by datapythonista — COMMENTED** [review]
Two small things, but changes look good. Thanks @dorothykiz1

File: setup.cfg [hunk 2]
```diff
@@ -50,4 +50,7 @@ exclude = setup.py,
         benchmarks/step_detect.py,
         benchmarks/__init__.py,
         docs/source/conf.py,
-        .pytest_cache
+        .pytest_cache,
+        ./asv/results.py,
+        ./asv/runner.py,
```
[the remark this answers is not in the document]
Comment (dorothykiz1) [reply setup.cfg]: Thanks for this highlight,let me fix it

**Comment by dorothykiz1** [comment]
Thank you, @datapythonista this PR is ready for your review again

**Pull request merged**